app/views/tutors/_form.html.erb

```diff
@@ -1,118 +1,186 @@
 
-  <% if @tutor.errors.any? %>
-    <div id="error_explanation">
-      <h2>
-        <%= pluralize(@tutor.errors.count, "error") %> prohibited
-        this tutor from being saved:
-      </h2>
-      <ul>
-        <% @tutor.errors.full_messages.each do |msg| %>
-          <li><%= msg %></li>
-        <% end %>
-      </ul>
-    </div>
-  <% end %>
-
-  <div class="row">
-    <div class="col-md-3">
-      <%= f.input :first_name %>
-    </div>
-    <div class="col-md-4">
-      <%= f.input :last_name %>
-    </div>
-    <div class="col-md-3">
-      <%= f.input :dob, as: :date, label: "Date of Birth",
-                                   start_year: Date.today.year - 70,
-                                   end_year: Date.today.year - 12 %>
+<% if @tutor.errors.any? %>
+  <div id="error_explanation">
+    <h2>
+      <%= pluralize(@tutor.errors.count, "error") %> prohibited
+      this tutor from being saved:
+    </h2>
+    <ul>
+      <% @tutor.errors.full_messages.each do |msg| %>
+        <li><%= msg %></li>
+      <% end %>
+    </ul>
+  </div>
+<% end %>
+
+<form role="form">
+
+  <div class="row">
+    <div class="col-md-6">
+      <!-- Affiliate will go here --!>
     </div>
-    <div class="col-md-2">
-      <div class="form-group">
-        <%= f.label "Gender" %><br>
-        <%= collection_radio_buttons(:tutor, :gender, [["male", "Male"] ,["female", "Female"]], :first, :last) do |b| %>
-          <label class="radio-inline">
-            <%= b.radio_button %> <%= b.text %>
-          </label>
-        <% end %>
+
+    <div class="col-md-6">
+      <div class="pull-right">
+        <label class="control-label" for="intakeDate"> Intake Date </label>
+        <input type="text" class="form-control datepicker" id="intakeDate" name="intake_date" placeholder="MM/DD/YYYY">
```

Comment:
Does this match the screenshot? Here it seems like it would be a text box but in your screenshot it shows 3 separate boxes for month day and year..